lib/preprocessors/generate-esdoc.js

```diff
@@ -25,7 +26,8 @@ const ESDOC_CONFIG = {
         dynamicImport: true
       }
     },
-    { name: 'esdoc-accessor-plugin' }
+    { name: 'esdoc-accessor-plugin' },
+    { name: 'esdoc-typescript-plugin', option: { enable: true } }
```

Comment:
```suggestion
    { name: 'esdoc-typescript-plugin' }
```

this `enable` option is `true` by default.